zerver/lib/actions.py

```diff
@@ -2997,3 +3005,34 @@ def make_dict(row):
     streams.sort(key=lambda elt: elt["name"])
 
     return streams
+
+def do_claim_attachment(message):
```

Comment:
this should be called `do_claim_attachments` since there may be more than one in the message